src/reanimated2/valueUnpacker.ts

```diff
@@ -74,6 +74,14 @@ type ValueUnpacker = WorkletFunction<
 >;
 
 if (__DEV__ && IS_NATIVE) {
+  const f = (() => {
+    'worklet';
+  }) as WorkletFunction<[], void>;
+  if (f.__workletHash === undefined) {
+    throw new Error(
+      `[Reanimated] Failed to create a worklet. See https://docs.swmansion.com/react-native-reanimated/docs/guides/troubleshooting#failed-to-create-a-worklet for more details.`
+    );
+  }
```

Comment:
I'm not sure if this is the file we want this check in. Is this file or `initializers.ts` called sooner?

Reply:
Looks like it is, hence the bug 😈

Reply:
:swiniok:

Reply:
But this check seems pretty unrelated to `valueUnpacker` - it's just a coincidence that lack of plugin failed here. I'd love to see this check be in a more meaningful place, but if you think it's okay here I will accept.

Reply:
Yes, it's unrelated to `valueUnpacker` but we need to check if worklet conversion was successful before we access `valueUnpacker.__closure` &ndash; otherwise, we get the error from the issue. I guess we could move the check to a separate file and call it from here but do we really need a separate file for this? Alternatively, we could move these two checks (1. workletHash, 2. valueUnpacker closure) to a new file but I think it's better to have valueUnpacker check in `valueUnpacker.ts`.

Reply:
🆗